src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/CloudException.cs

```diff
@@ -55,5 +55,10 @@ public CloudException(string message) : base(message)
         public CloudException(string message, Exception innerException) : base(message, innerException)
         {
         }
+
+        /// <summary>
+        /// Gets information about the HTTP status code returned.
+        /// </summary>
+        public int HttpStatusCode { get; set; }
```

Comment:
Any reason we want this added
Most of the time especially in LRO, your HttpStatus code will be 200 but the operation fails for other reasons.
StatusCode is available on the Response property.
Is this just for the sake of convenience or is there more to it?

Reply:
1. Convenience. 
2. Exception messages earlier used to specify the status code in the message itself which may change with new changes, so it makes sense to have this available handy.